README.md

```diff
@@ -1,2 +1,83 @@
-# wallet
-A budget handler application
+# Wallet Application 
+
+
+## Database deploy
+
+Database is provisioned using `docker-compose`. For that, following steps must be accomplished:
+
+1. Set wallet's database password by creating following environmental variable
+
+   ```bash
+   $ export MYSQL_PASSWD=<password>
+   ```
+
+   
+
+2. Set MySQL's root password by creating following environmental variable
+
+   ```bash
+   $ export MYSQL_ROOT_PASSWD=<password>
+   ```
+
+   
+
+3. Deploy (here we are in base dir)
+
+   ```bash
+   $ docker-compose -f provision/mariadb.yml up
+   ```
+
+   
+
+## Database connectivity - CLI
+
+### From host
+
+In order to test database connectivity from host, we need to use `mysql-client`, and run following command:
```

Comment:
Looks like we need to install an app on our system after all, which isn't great. It would be better to just do manage the database through the docker container directly instead of polluting the host system with additional software (the host should really only need docker).

Reply:
We're not going to be connecting with this client from the host, so we should delete this section from the README.